XLForm/XL/Cell/XLFormBaseCell.h

```diff
@@ -35,7 +35,7 @@
 
 @property (nonatomic, weak) XLFormRowDescriptor * rowDescriptor;
 
--(XLFormViewController *)formViewController;
+@property (nonatomic, readonly, strong) XLFormViewController *formViewController;
```

Comment:
@jkereako but it also allocates memory to the property state.  `formViewController ` method is much more than a property.  `formViewController ` doesn't need memory to allocate any state and the implementation is much more than a simple variable assignment. This is my point.